kani-driver/src/call_goto_instrument.rs

```diff
@@ -37,15 +37,15 @@ impl KaniSession {
             self.goto_sanity_check(output)?;
         }
 
-        self.instrument_contracts(harness, output)?;
-
         if self
             .args
             .common_args
             .unstable_features
             .contains(kani_metadata::UnstableFeature::LoopContracts)
         {
-            self.instrument_loop_contracts(harness, output)?;
+            self.instrument_contracts_with_loop_contracts(harness, output)?;
+        } else {
+            self.instrument_contracts(harness, output)?;
```

Comment:
Can you merge these functions and take a boolean on whether to initialize loop contracts?